lib/widget/youtube/youtubestate.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:ensemble/widget/youtube/native/youtube_platform_native.dart'
+    if (dart.library.html) './web/youtube_platform_web.dart';
+
+abstract class YoutubeWeb {
+  factory YoutubeWeb() => getInstance();
+  void createWebInstance();
```

Comment:
it doesn't make sense to call it createWebInstance when the native has to implement it as well. 
Similarly it does not make sense to call this abstract class YoutubeWeb when you know native class has to implement it as well. 

Just call it YouTubeBase

Reply:
@fullstackctp think about using export instead of import and have a library, look at webview as an example. It seems cleaner, what do you think?